dashboard/app/assets/stylesheets/lti/early_access_banner.scss

```diff
@@ -0,0 +1,31 @@
+@import 'color';
+@import 'font';
+
+#lti-early-access-banner {
```

Comment:
Consider having the banner default to hidden and then shown by Javascript if we should show it.
It would be annoying to dismiss the banner but then you see it flash every time you load a new page.

Reply:
https://github.com/code-dot-org/code-dot-org/blob/94e30abff56d4a975b2dde9ec649fc9c27ea8126/dashboard/app/views/lti/v1/_early_access_banner.html.haml#L1